docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport.js integration module
+
+{WFM-RC-NameShort} authentication and authorisation module using link:http://passportjs.org[PassportJS].
+The Passport.js module is the default security module for {WFM-RC-NameShort}.
+
+Authentication mechanisms, known as link:http://passportjs.org/docs/configure[strategies], are packaged as individual modules.
```

Comment:
Basically we saying this before mentioning strategies bellow. We can move down or remove.